Dockerfile

```diff
@@ -6,7 +6,7 @@ ENV GOPATH /go
 ENV PATH $GOROOT/bin:$PATH
 ENV APPLOC $GOPATH/src/github.com/digitalocean/ceph_exporter
 
-RUN apt-get install -y apt-transport-https
+RUN apt-get update && apt-get install -y apt-transport-https
```

Comment:
Very interesting. I don't see it somewhy. It shouldn't hurt to have this one in then.